### PR TITLE
New PDA and Cartridges Icons

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

After doing the new IDs sprites, I wasn't really happy on how the PDAs where looking kinda clashing, so I decided to update the PDAs, alongside their cartridges that haven't really been updated since their inception way back in time.

Also fixed the exploration crew PDA/ID being unavailable from the ID painter machine

## Why It's Good For The Game

New icons good and also give a better idea to what someone might look at first glance when they see someone's PDA on the ground.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

superiorIcons

</details>

## Changelog
:cl:
add: now Syndicate PDAs cannot be detonated anymore
fix: tweaked numerous description of many PDAs, like the Captain's
fix: the VIP now get their proper full golden ID to demonstrate their wealth
fix: fixed the missing option to paint a PDA/ID with the Exploration crew's colours
imageadd: Added new icons for the PDAs and cartridges
/:cl:

### CODE REVIEW

**Comment by PestoVerde322** [comment]
**_NOTE FOR MAINTAINERS:_** This may require some testing before a full merge, suggesting a Testmerge before a full on merge

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**
